modules/cluster-logging-collector-log-forwarding-supported-plugins-5-4.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-log-forwarding-supported-plugins-5-4_{context}"]
+
+= Supported log data output types in OpenShift Logging 5.4
+
+Red Hat OpenShift Logging 5.4 provides the following output types and protocols for sending log data to target log collectors.
+
+Red Hat tests each of the combinations shown in the following table. However, you should be able to send log data to a wider range target log collectors that ingest these protocols.
+
+[options="header"]
+|====
+| Output types   | Protocols          | Tested with
+
+| Amazon CloudWatch
+| REST over HTTPS
+| The current version of Amazon CloudWatch
+
+
+| elasticsearch
+| elasticsearch
+a| Elasticsearch 7.10.1
+
+| fluentdForward
+| fluentd forward v1
+a| fluentd 1.14.5
+
+logstash 7.10.1
+
+| Loki
+| REST over HTTP and HTTPS
+| Loki 2.2.1 deployed on OCP
+
+| kafka
+| kafka 0.11
+a| kafka 2.7.0
+
+| syslog
+| RFC-3164, RFC-5424
+| rsyslog-8.39.0
+
+|====
```

Comment:
The fluentd doesn't support Elasticsearch 8  now(5.6.2)
The vector doesn't support  fluentd/logstash/rsyslog now(5.6.2)
Can we provide a note?  Can we make two tables for each colllector type?

Reply:
I think a note to that effect and maybe a link pointing out the future deprecation of fluentd would be sufficient and would help keep the docs concise. 

After all, the text above the tables merely says that these are the protocols, and collectors we test against so _technically_ it could be argued that the table is accurate like this. What do you think?

Reply:
hi @anpingli just wanted to follow up on my last comment